pkg/controller/nodenetworkstate/nodenetworkstate_controller.go

```diff
@@ -24,9 +26,21 @@ import (
 
 var (
 	log                     = logf.Log.WithName("controller_nodenetworkstate")
-	nodenetworkstateRefresh = 5 * time.Second
+	nodenetworkstateRefresh time.Duration
 )
 
+func init() {
+	refreshTime, isSet := os.LookupEnv("NODE_NETWORK_STATE_REFRESH_INTERVAL")
+	if !isSet {
+		panic("NODE_NETWORK_STATE_REFRESH_INTERVAL is mandatory")
+	}
+	intRefreshTime, err := strconv.Atoi(refreshTime)
```

Comment:
Can't we verify that at configmap type ?

Reply:
ConfigMap values are only string.